## LAB__2a

### 1 Виконав усі базові приклади у файлі test.py

### 2  Результат виконання команди python .
``````shell
2021-10-04 12:26:58.666775
win32
test
``````
### 3  Результат виконання команди  python . -h 
```shell
usage: . [-h] [-o OPT] [-l]

Приклад передачі аргументів у Python програму.

optional arguments:
  -h, --help            show this help message and exit
  -o OPT, --optional OPT
                        Цей параметр є вибірковим.
  -l, --logs            Якщо виконати команду з цим параметром будуть виводитись логи.
```
### 4 Передав параметр за допомогою python . -o "Цей текст також має вивестись"
```shell
We are in the __main__
2021-10-04 12:33:22.105716
win32
З консолі було передано аргумент
 ========== >> Цей текст також має вивестись << ==========
test

```
### 5  Запустивши з консолі python . --logs були виведені  логи
```shell
2021-10-04 12:32:05,096 root INFO: Тут буде просто інформативне повідомлення
2021-10-04 12:32:05,096 root WARNING: Це Warning повідомлення
2021-10-04 12:32:05,112 root ERROR: Це повідомлення про помилку
test

```
### 6 Створив власну функцію 
```shell
def EvenOdd(bool):
    for i in range(0,101):
        if(bool and i % 2 == 0):
            print(i)
        elif(not bool and i % 2 != 0):
            print(i)
```
### 7 Створив функцію з помилкою, у випадку її виникнення виводиться повідомлення про помилку за допомогою логування
```shell
def error(x,y):
    try:
        a = x/y
        log.info("Програма виконалась")
        return a
    except Exception as ERROR:
        log.error("Помилка")
```
   ### common.error(15,1)
```shell
2021-10-04 13:07:33,028 root INFO: Програма виконалась
```

### 8
Створив таблицю і пул реквест

